gcp/modules/couchdb/values.yaml

```diff
@@ -27,6 +27,8 @@ couchdbConfig:
     bind_address: any
     # http://docs.couchdb.org/en/stable/config/auth.html#authentication-configuration
     require_valid_user: true
+  log:
+    level: warn
```

Comment:
Does this mean that we're not going to see actual HTTP requests to CouchDB anymore in logs? If it is the case, I think this is undesirable.

Reply:
I understand, by the documentation, that we would only remove the 2XX requests logs, other requests as 5XX will be logged. http://docs.couchdb.org/en/stable/config/logging.html#log/level

Do you think that we should save ALL the HTTP requests? I think that those logs would demand more resources than the real value of that information.

Reply:
Replied in thread.